### PR TITLE
Set up Grants model in Express

### LINKED ISSUE #389: Port grants data modeling work from Django to Express API
Extensive work was done on getting data modeling set up in Django. We want to leverage this by porting this work to the API backend.

**We'll know we're done when:**
- [x] API connects to the Postgres database
  - [x] Local
  - [x] Sandbox
- [x] API endpoint for `/grants` model is returning minimal grants JSON from the database
- [x] Display all grants model data for the API endpoint (no relationships, creation, editing, validation)
  - [x] Strongly-typed (types for option choices are in place)
  - [x] Verify `decimal` type parameters are translated correctly
- [x] Frontend app displays a few of these grants

### PR DESCRIPTION
**Pull request by beechnut**
Uses Sequelize ORM to model Grants.

Connects to Postgres database. Tested in local (docker-compose) and sandbox AWS environments.

Closes #389

### CODE REVIEW

File: api/src/models.ts [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+  public wpapStatus!: WpapStatusChoices | null;
+  public wpapStatusDate!: Date;
+  public cooperatorAgreementNumber!: string;
+  public gid!: string;
+  public adminOpen!: string;
+  public lastUpdate!: Date;
+}
+
+/**
+ * When copying from the spreadsheet, remember to:
```
Comment (neilmb) [inline api/src/models.ts]: Do we want to put a link for the spreadsheet that you're referring to here? Otherwise this seems sort of ambiguous.

File: decisions/architecture/000-record-architecture-decisions-in-the-repository.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
 # 0. Record architecture decisions in the repository.
```
Comment (neilmb) [inline decisions/architecture/000-record-architecture-decisions-in-the-repository.md]: This feels like an unrelated change, did it come in from a different branch/PR?

File: frontend/src/app/grant.ts [hunk 3]
```diff
@@ -1,20 +1,129 @@
-enum Status {
-  None = '---',
-  One = '01',
-  Two = '02',
-}
+type AbChoices = '----' | '01' | '02';
```
Comment (neilmb) [inline frontend/src/app/grant.ts:1]: **thought:** Just another place that it will be nice to re-use these between front- and backend.

File: nrm_django/grants/models.py [hunk 4]
```diff
@@ -58,7 +58,10 @@ class Grant(models.Model):
     )
     # Sys-generated, once initial proposal saved. Control number.
     # Older field, not user viewable. Only used for historical records.
-    application_id = models.CharField(max_length=34, editable=False)
+    application_id = models.CharField(
```
Comment (neilmb) [inline nrm_django/grants/models.py]: **suggestion (non-blocking):** I'm surprised to see changes in `nrm_django`. These seem like auto-formatting changes? Could we leave these changes out to make the PR better focused?

File: package.json [hunk 5]
```diff
@@ -7,5 +7,8 @@
   },
   "scripts": {
     "test:a11y": "pa11y-ci --config a11y/scripts/main.json"
+  },
+  "dependencies": {
+    "prettier": "^2.3.2"
```
Comment (neilmb) [inline package.json:12]: I'm not sure what the top-level `package.json` file is for here. If it isn't in the `api/` or `frontend/` directories, then what depends on prettier here? Is it for the CI setup?

**Review by neilmb — DISMISSED** [review]

File: frontend/src/app/grant.ts (hunk 3 above)
Comment (beechnut) [reply frontend/src/app/grant.ts:1]: Agreed. Types and interfaces are the places for this to happen.

File: nrm_django/grants/models.py (hunk 4 above)
Comment (beechnut) [reply nrm_django/grants/models.py]: Thanks for catching—this was not intentional!

File: decisions/architecture/000-record-architecture-decisions-in-the-repository.md (hunk 2 above)
Comment (beechnut) [reply decisions/architecture/000-record-architecture-decisions-in-the-repository.md]: Thanks for catching this, will fix.

File: package.json (hunk 5 above)
Comment (beechnut) [reply package.json:12]: > Is it for the CI setup?

Yes. Package management has been a little weird, but I figured a library that affects the whole project's CI should go in the root directory, especially when it's not a dependency for either `api` or `frontend`.

File: api/src/models.ts (hunk 1 above)
Comment (beechnut) [reply api/src/models.ts]: I guess it's a little confusing to reference a private spreadsheet in a public repo. We'll leave it out for now.

**Review by neilmb — APPROVED** [review]
